lazylint

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Thanks for this, what can we do with it? 😅

Reply:
Oh! Oops.

Reply:
It does pretty much the same as the core one. Runs the linters on any file that's changed from dev.